### PR TITLE
Add function and tests to reveal cards

### PR DESCRIPTION
**Pull request by vbeardow**
Added tests and methods to actions.py to reveal cards. Either reveal one card or reveal all cards by setting reveal_all = true/false.

Fixed some linting issues using pylint. 

Use fixtures across multiple tests by adding them to conftest and setting the scope.

### CODE REVIEW

File: blackjack/actions.py [hunk 1]
```diff
@@ -31,3 +32,19 @@ def stick_or_twist(hand: Hand, deck: Deck) -> None:
         else:
             raise Exception("Please enter either s or t")
         break
+
+
+def reveal_cards(my_hand, reveal_all=True) -> None:
+    """Reveal cards in the hand, printing the rank and value of the card
+
+    Args:
+        my_hand (_type_): Hand object representing a hand of cards
```
Comment (Edward-Boulderstone) [inline blackjack/actions.py]: What does __type__ mean?

File: tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import pytest
+from blackjack.hand import Hand
+from blackjack.deck import Deck
+from blackjack.chips import Chips
+
+
+@pytest.fixture(scope="module")
+def my_hand() -> Hand:
```
Comment (Edward-Boulderstone) [inline tests/conftest.py:8]: Is it worth having a fixture for a default initializer?

**Review by Edward-Boulderstone — APPROVED** [review]

File: blackjack/actions.py (hunk 1 above)
Comment (vbeardow) [reply blackjack/actions.py]: Just missed out a type hint there - good spot!

File: tests/conftest.py (hunk 2 above)
Comment (vbeardow) [reply tests/conftest.py:8]: The idea behind was to avoid initialise the object I use for testing every time I run a new test that uses the same object.

Comment (Edward-Boulderstone) [reply tests/conftest.py:8]: That makes sense

File: blackjack/actions.py (hunk 1 above)
Comment (vbeardow) [reply blackjack/actions.py]: Updated

**Pull request merged**